src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -416,17 +434,22 @@ export default React.memo(function GamePage(): JSX.Element | null {
             {/* NEW DESIGN */}
             {experimentalFeatures.enableNewDesign && (
               <>
+                <div className="topRowWrapper">
+                  <div className="opts opts-left">
+                    <NavLink
+                      className="backButton"
+                      to={backRoute}
+                      title={t2('webview.controls.back', 'Go Back')}
+                    >
+                      <ArrowBackIosNew />
+                    </NavLink>
+                  </div>
+                  <div className="opts opts-right">
+                    {!isBrowserGame && <SettingsButton gameInfo={gameInfo} />}
+                    <DotsMenu gameInfo={gameInfo} handleUpdate={handleUpdate} />
+                  </div>
+                </div>
```

Comment:
a bit nit picky, but we can avoid using the `opts` wrappers if we have the 3 elements inside `topRowWrapper` and then you set `.topRowWrapper .backButton { margin-inline-end: auto }`

(you'd need to move the gap from `.opts` to `.topRowWrapper`)

Reply:
Fixed